nerdlandbot/commands/GuildData.py

```diff
@@ -19,6 +19,9 @@ class GuildData:
     youtube_channels: dict
     purgers: dict
     culture: str
+    pets: dict
+    pets_last_id: int
+    pets_categories: list
```

Comment:
It'd be nice if the dicts were typed as well, though unfortunately the other stuff isn't either. Though perhaps it's good to document what each field does and what its format is if it's not typed.

```suggestion
    pets_categories: List[str]
```